doc/mantohtml.c

```diff
@@ -1161,7 +1161,7 @@ html_fputs(const char *s,		/* I  - String */
         html_putc(*s++, fp);
       }
     }
-    else if (!strncmp(s, "http://", 7) || !strncmp(s, "https://", 8) || !strncmp(s, "ftp://", 6))
+    else if ((!strncmp(s, "http://", 7) || !strncmp(s, "https://", 8) || !strncmp(s, "ftp://", 6)) && !strnstr(s, ".local", strlen(s)))
```

Comment:
strnstr is non-portable. I'll come up with a more portable replacement.

Reply:
Upon reflection, neither strnstr() nor strstr() are good solutions because it would fail to produce a link to any URL with ".local" in them. We want just the last domain element.

Reply:
I have some replacement code for strnstr() (another pull request coming). We could consider using uriparser (https://uriparser.github.io) if that isn't good enough for us down the road.

Reply:
Smith,

First, libcups already has a perfectly good URL decoder, and second I don’t think we need to go that far for this program - it is just for converting the ippsample man pages after all... :)

Reply:
Agreed. I didn't look in libcups but upon reflection I figured it had to have something in there. But before that I saw your changes and they are fine - good enough for here. I'm going to move on.